internal/server/v1/event/event.go

```diff
@@ -27,6 +30,27 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+type filterSpec struct {
+	prop  string
+	unit  string
+	limit float64
+}
+
+func keepEvent(event *pb.EventCollection_Event, spec filterSpec) bool {
+	for prop, vals := range event.GetPropVals() {
+		if prop == spec.prop {
+			valStr := strings.TrimPrefix(vals.Vals[0], spec.unit)
```

Comment:
just checking, will the unit always be the prefix?

Reply:
It should be from current schema. If it deviates, we can set up a config here so the function is based on the unit